swatch-producer-aws/deploy/clowdapp.yaml

```diff
@@ -41,8 +41,6 @@ parameters:
     value: '3'
   - name: SPLUNK_HEC_INCLUDE_EX
     value: 'true'
-  - name: SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT
```

Comment:
Do you mind putting an MR out in app-interface to clean up `SWATCH_INTERNAL_SUBSCRIPTION_ENDPOINT`  out of the [deploy-clowder.yml](https://gitlab.cee.redhat.com/service/app-interface/-/blob/78aa92465a4e3afbf1a44e28563b42fe60b1d062/data/services/insights/rhsm/cicd/deploy-clowder.yml#L137) since it's no longer being exposed to override via env var?

It actually looks like it can be removed out of the `swatch-producer-azure` of that same file too.

Reply:
Done: https://gitlab.cee.redhat.com/service/app-interface/-/merge_requests/88459